app/models/game.rb

```diff
@@ -1,4 +1,5 @@
 class Game < ActiveRecord::Base
   validates :name, presence: true
+  scope :available, -> {where(white_player_id == nil || black_player_id == nil)}
```

Comment:
My goal with this is. The way the scope is written currently won't work. Writing a test will show you :).